### PR TITLE
Market - configurable (ENV) offer expiration (to enable tests)

### PR DESCRIPTION
**Pull request by nieznanysprawiciel**
Necessary for yapapi tests in goth.
Env var:
`DEFAULT_SUBSCRIPTION_TTL`

### CODE REVIEW

File: core/market/src/config.rs [hunk 1]
```diff
@@ -23,7 +23,9 @@ pub struct DiscoveryConfig {
     pub mean_cyclic_unsubscribes_interval: Duration,
 }
 
+#[derive(StructOpt)]
 pub struct SubscriptionConfig {
+    #[structopt(env, parse(try_from_str = parse_chrono_duration), default_value = "1h")]
```
Comment (azawlocki) [inline core/market/src/config.rs:28]: I'd call the variable `DEFAULT_SUBSCRIPTION_TTL` or `SUBSCRIPTION_DEFAULT_TTL` (whichever fits `yagna` naming convention for config env vars better).

**Review by azawlocki — COMMENTED** [review]
I've left some suggestions for a more informative name for the environment variable.

**Review by azawlocki — APPROVED** [review]
LGTM!

**Comment by nieznanysprawiciel** [comment]
Related to https://github.com/golemfactory/yagna-triage/issues/95

**Review by tworec — APPROVED** [review]

**Pull request merged**